app/models/consultation.rb

```diff
@@ -151,7 +161,7 @@ def send_consultee_emails(attributes)
   end
 
   def start_deadline(now = Time.zone.now)
-    update!(end_date: [end_date, end_date_from(now)].compact.max, start_date: start_date || default_start_date(now))
+    update!(end_date: [end_date_from(now), consultee_response_required_by(now)].compact.max, start_date: start_date || default_start_date(now))
```

Comment:
This behaviour isn't quite the same as before. The previous version is the maximum of the current end date or the new end date; i.e., it won't be updated if the new end date is earlier than the existing one. But it looks like the new version will always change it?

Ah, but `consultee_response_required_by` will always be later than today. That feels very similar to `end_date_from` though. Can't we just use `end_date_from` in this case? (i.e., leave this line as it was?)